Lib/test/test_readline.py

```diff
@@ -145,11 +141,11 @@ def test_init(self):
 """
 
     def test_auto_history_enabled(self):
-        output = run_pty(self.auto_history_script.format(True))
+        rc, output = run_pty(self.auto_history_script.format(True))
         self.assertIn(b"History length: 1\r\n", output)
```

Comment:
You may add self.assertEqual(rc, 0) in these tests **after** checking the output.